### DISCUSSION TITLE
Is there a way to identify and remove flattened highlights?

### DISCUSSION THREAD

**Discussion by dummifiedme** [pymupdf/PyMuPDF#1676] [Looking for help] [answered]
I have several pdfs with highlighted text which my friends annotated using GoodNotes and the generated pdfs have all the highlights flattened (non deletable or editable).

Is there a way to identify and remove them using pymupdf?

**Comment by JorjMcKie** [comment, marked as answer]
You (probably) cannot remove them: as you describe it, the annotations have been converted to normal, permanent page content. So they have become regular text and / or drawings, and / or potentially images.
While you are able to delete text (using redaction annotations) and images if you can properly identify the regions on the page, the same is not possible for drawings.

**Reply by dummifiedme** [reply]
That's what I feared. I guess, I will live with it. No problems :D
